OpenAPI/error.json

```diff
@@ -0,0 +1,14 @@
+{
+    "type" : "object",
+    "properties" : {
+        "title": {
```

Comment:
properties for error:
`ok` - false in case of any error
`errorMessage` - short error description